services/core/fuse/config.yaml

```diff
@@ -1,5 +1,8 @@
 service:
   name: fuse
+  # domain is the domain name of the whole draft cluster that is running. (ie. draft.com)
+  # This is used to generate the urls for the processes that register themselves with `fuse`
+  domain: localhost
```

Comment:
we should probably create a separate parent key called `routing` or something like that to avoid confusion as `domain` could also refer to the draft domain of the process

Reply:
I see yeah. I was thinking about the domain the services would be available on. What about `domain_url`?

Reply:
Can you move it to a different parent? So maybe:
```yaml
routing:
    domain: localhost
````